src/components/ResultsListItem.vue

```diff
@@ -5,11 +5,13 @@
     <p>This park is {{ result.opening_hours.open_now ? 'open' : 'closed' }}</p>
     <p>Rating: {{ result.rating }}</p>
     <section class="item-card-bottom">
-      <button v-if="this.$route.path !== '/my-parks'" @click="savePark" class="save-button">
-        SAVE
-      </button>
-      <button v-else @click="unsavePark" class="unsave-button">
-        UNSAVE
+      <button 
+        v-if="this.$route.path !== '/my-parks'" 
```

Comment:
Good call here with this `v-if` to make sure we don't render a `save` button when we're on the `my-parks` route

Reply:
You got it!